CI error: Unknown system error -88

https://github.com/lfortran/lfortran/actions/runs/14273988761/job/40013300515

```
ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:25.9176410Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:25.9277680Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:25.9278000Z [ 98%] Built target example_math_arg
2025-04-04T21:37:25.9378320Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:25.9384800Z [ 98%] Built target example_math_argpi
2025-04-04T21:37:25.9481880Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:25.9482260Z [ 98%] Built target example_math_argd
2025-04-04T21:37:25.9565480Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:25.9676050Z [ 98%] Linking Fortran executable example_math_is_close
2025-04-04T21:37:25.9785000Z [ 99%] Building Fortran object example/stats/CMakeFiles/example_mean.dir/example_mean.f90.o
2025-04-04T21:37:25.9886270Z [ 99%] Linking Fortran executable example_math_arange
2025-04-04T21:37:26.0288500Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.0389540Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.0491090Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.0491440Z [ 99%] Built target example_math_is_close
2025-04-04T21:37:26.0592310Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.0592620Z [ 99%] Built target example_math_arange
2025-04-04T21:37:26.0990330Z [ 99%] Linking Fortran executable example_mean
2025-04-04T21:37:26.1536220Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.1637360Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.1838110Z [ 99%] Built target example_mean
2025-04-04T21:37:26.4031180Z [ 99%] Linking Fortran executable test_linalg_matrix_property_checks
2025-04-04T21:37:26.4532060Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.4542770Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:26.4656340Z [ 99%] Built target test_linalg_matrix_property_checks
2025-04-04T21:37:26.9962210Z [100%] Linking Fortran executable test_stdlib_math
2025-04-04T21:37:27.0370140Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:27.0370680Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:27.0522450Z [100%] Built target test_stdlib_math
2025-04-04T21:37:32.8711420Z [100%] Linking Fortran executable test_selection
2025-04-04T21:37:32.9096820Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:32.9097450Z ld: warning: duplicate -rpath '/Users/runner/micromamba/envs/lf/lib' ignored
2025-04-04T21:37:32.9255960Z [100%] Built target test_selection
2025-04-04T21:37:32.9384470Z + ctest
2025-04-04T21:37:32.9481700Z Test project /Users/runner/work/lfortran/lfortran/stdlib
2025-04-04T21:37:32.9521540Z       Start  1: logicalloc
2025-04-04T21:37:32.9523600Z Process not started
2025-04-04T21:37:32.9523930Z  /Users/runner/work/lfortran/lfortran/stdlib/test/array/test_logicalloc
2025-04-04T21:37:32.9524230Z [Unknown system error -88]
2025-04-04T21:37:32.9525750Z  1/90 Test  #1: logicalloc .......................***Not Run   0.00 sec
2025-04-04T21:37:32.9527330Z       Start  2: ascii
2025-04-04T21:37:32.9528980Z Process not started
2025-04-04T21:37:32.9529330Z  /Users/runner/work/lfortran/lfortran/stdlib/test/ascii/test_ascii
2025-04-04T21:37:32.9529580Z [Unknown system error -88]
2025-04-04T21:37:32.9530750Z  2/90 Test  #2: ascii ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9531620Z       Start  3: loadtxt
2025-04-04T21:37:32.9533290Z Process not started
2025-04-04T21:37:32.9533620Z  /Users/runner/work/lfortran/lfortran/stdlib/test/io/test_loadtxt
2025-04-04T21:37:32.9533870Z [Unknown system error -88]
2025-04-04T21:37:32.9535330Z  3/90 Test  #3: loadtxt ..........................***Not Run   0.00 sec
2025-04-04T21:37:32.9536400Z       Start  4: loadtxt_qp
2025-04-04T21:37:32.9537990Z Process not started
2025-04-04T21:37:32.9538360Z  /Users/runner/work/lfortran/lfortran/stdlib/test/io/test_loadtxt_qp
2025-04-04T21:37:32.9538610Z [Unknown system error -88]
2025-04-04T21:37:32.9539970Z  4/90 Test  #4: loadtxt_qp .......................***Not Run   0.00 sec
2025-04-04T21:37:32.9540990Z       Start  5: savetxt_qp
2025-04-04T21:37:32.9542570Z Process not started
2025-04-04T21:37:32.9542910Z  /Users/runner/work/lfortran/lfortran/stdlib/test/io/test_savetxt_qp
2025-04-04T21:37:32.9543160Z [Unknown system error -88]
2025-04-04T21:37:32.9544290Z  5/90 Test  #5: savetxt_qp .......................***Not Run   0.00 sec
2025-04-04T21:37:32.9545190Z       Start  6: parse_mode
2025-04-04T21:37:32.9546880Z Process not started
2025-04-04T21:37:32.9547220Z  /Users/runner/work/lfortran/lfortran/stdlib/test/io/test_parse_mode
2025-04-04T21:37:32.9547480Z [Unknown system error -88]
2025-04-04T21:37:32.9548570Z  6/90 Test  #6: parse_mode .......................***Not Run   0.00 sec
2025-04-04T21:37:32.9549490Z       Start  7: linalg
2025-04-04T21:37:32.9551170Z Process not started
2025-04-04T21:37:32.9551650Z  /Users/runner/work/lfortran/lfortran/stdlib/test/linalg/test_linalg
2025-04-04T21:37:32.9551910Z [Unknown system error -88]
2025-04-04T21:37:32.9552870Z  7/90 Test  #7: linalg ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9553810Z       Start  8: linalg_matrix_property_checks
2025-04-04T21:37:32.9555440Z Process not started
2025-04-04T21:37:32.9555730Z  /Users/runner/work/lfortran/lfortran/stdlib/test/linalg/test_linalg_matrix_property_checks
2025-04-04T21:37:32.9556060Z [Unknown system error -88]
2025-04-04T21:37:32.9557090Z  8/90 Test  #8: linalg_matrix_property_checks ....***Not Run   0.00 sec
2025-04-04T21:37:32.9558040Z       Start  9: optval
2025-04-04T21:37:32.9559600Z Process not started
2025-04-04T21:37:32.9559890Z  /Users/runner/work/lfortran/lfortran/stdlib/test/optval/test_optval
2025-04-04T21:37:32.9560150Z [Unknown system error -88]
2025-04-04T21:37:32.9561140Z  9/90 Test  #9: optval ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9561880Z       Start 10: selection
2025-04-04T21:37:32.9563360Z Process not started
2025-04-04T21:37:32.9563710Z  /Users/runner/work/lfortran/lfortran/stdlib/test/selection/test_selection
2025-04-04T21:37:32.9563980Z [Unknown system error -88]
2025-04-04T21:37:32.9564860Z 10/90 Test #10: selection ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9565710Z       Start 11: specialfunctions_gamma
2025-04-04T21:37:32.9567100Z Process not started
2025-04-04T21:37:32.9567490Z  /Users/runner/work/lfortran/lfortran/stdlib/test/specialfunctions/test_specialfunctions_gamma
2025-04-04T21:37:32.9567830Z [Unknown system error -88]
2025-04-04T21:37:32.9568760Z 11/90 Test #11: specialfunctions_gamma ...........***Not Run   0.00 sec
2025-04-04T21:37:32.9569550Z       Start 12: string_assignment
2025-04-04T21:37:32.9570980Z Process not started
2025-04-04T21:37:32.9571320Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_assignment
2025-04-04T21:37:32.9572480Z [Unknown system error -88]
2025-04-04T21:37:32.9572700Z 12/90 Test #12: string_assignment ................***Not Run   0.00 sec
2025-04-04T21:37:32.9573240Z       Start 13: string_operator
2025-04-04T21:37:32.9574730Z Process not started
2025-04-04T21:37:32.9574980Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_operator
2025-04-04T21:37:32.9575250Z [Unknown system error -88]
2025-04-04T21:37:32.9576340Z 13/90 Test #13: string_operator ..................***Not Run   0.00 sec
2025-04-04T21:37:32.9577110Z       Start 14: string_intrinsic
2025-04-04T21:37:32.9578720Z Process not started
2025-04-04T21:37:32.9578970Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_intrinsic
2025-04-04T21:37:32.9579250Z [Unknown system error -88]
2025-04-04T21:37:32.9580280Z 14/90 Test #14: string_intrinsic .................***Not Run   0.00 sec
2025-04-04T21:37:32.9580970Z       Start 15: string_match
2025-04-04T21:37:32.9582540Z Process not started
2025-04-04T21:37:32.9582850Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_match
2025-04-04T21:37:32.9584070Z [Unknown system error -88]
2025-04-04T21:37:32.9584290Z 15/90 Test #15: string_match .....................***Not Run   0.00 sec
2025-04-04T21:37:32.9584870Z       Start 16: string_functions
2025-04-04T21:37:32.9586500Z Process not started
2025-04-04T21:37:32.9586740Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_functions
2025-04-04T21:37:32.9587020Z [Unknown system error -88]
2025-04-04T21:37:32.9588100Z 16/90 Test #16: string_functions .................***Not Run   0.00 sec
2025-04-04T21:37:32.9588890Z       Start 17: string_strip_chomp
2025-04-04T21:37:32.9590480Z Process not started
2025-04-04T21:37:32.9590830Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_strip_chomp
2025-04-04T21:37:32.9591110Z [Unknown system error -88]
2025-04-04T21:37:32.9592090Z 17/90 Test #17: string_strip_chomp ...............***Not Run   0.00 sec
2025-04-04T21:37:32.9592830Z       Start 18: string_to_string
2025-04-04T21:37:32.9594360Z Process not started
2025-04-04T21:37:32.9594690Z  /Users/runner/work/lfortran/lfortran/stdlib/test/string/test_string_to_string
2025-04-04T21:37:32.9594970Z [Unknown system error -88]
2025-04-04T21:37:32.9595880Z 18/90 Test #18: string_to_string .................***Not Run   0.00 sec
2025-04-04T21:37:32.9596650Z       Start 19: trapz
2025-04-04T21:37:32.9598100Z Process not started
2025-04-04T21:37:32.9598380Z  /Users/runner/work/lfortran/lfortran/stdlib/test/quadrature/test_trapz
2025-04-04T21:37:32.9598650Z [Unknown system error -88]
2025-04-04T21:37:32.9599680Z 19/90 Test #19: trapz ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9600430Z       Start 20: simps
2025-04-04T21:37:32.9601880Z Process not started
2025-04-04T21:37:32.9602190Z  /Users/runner/work/lfortran/lfortran/stdlib/test/quadrature/test_simps
2025-04-04T21:37:32.9602460Z [Unknown system error -88]
2025-04-04T21:37:32.9603430Z 20/90 Test #20: simps ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9604170Z       Start 21: gauss
2025-04-04T21:37:32.9605650Z Process not started
2025-04-04T21:37:32.9605960Z  /Users/runner/work/lfortran/lfortran/stdlib/test/quadrature/test_gauss
2025-04-04T21:37:32.9606210Z [Unknown system error -88]
2025-04-04T21:37:32.9607190Z 21/90 Test #21: gauss ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9607920Z       Start 22: stdlib_math
2025-04-04T21:37:32.9609500Z Process not started
2025-04-04T21:37:32.9609800Z  /Users/runner/work/lfortran/lfortran/stdlib/test/math/test_stdlib_math
2025-04-04T21:37:32.9610060Z [Unknown system error -88]
2025-04-04T21:37:32.9611020Z 22/90 Test #22: stdlib_math ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9611800Z       Start 23: linspace
2025-04-04T21:37:32.9613350Z Process not started
2025-04-04T21:37:32.9613650Z  /Users/runner/work/lfortran/lfortran/stdlib/test/math/test_linspace
2025-04-04T21:37:32.9613920Z [Unknown system error -88]
2025-04-04T21:37:32.9614950Z 23/90 Test #23: linspace .........................***Not Run   0.00 sec
2025-04-04T21:37:32.9615790Z       Start 24: logspace
2025-04-04T21:37:32.9617350Z Process not started
2025-04-04T21:37:32.9617680Z  /Users/runner/work/lfortran/lfortran/stdlib/test/math/test_logspace
2025-04-04T21:37:32.9617940Z [Unknown system error -88]
2025-04-04T21:37:32.9618860Z 24/90 Test #24: logspace .........................***Not Run   0.00 sec
2025-04-04T21:37:32.9619550Z       Start 25: gamma
2025-04-04T21:37:32.9621150Z Process not started
2025-04-04T21:37:32.9621560Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_gamma
2025-04-04T21:37:32.9621880Z [Unknown system error -88]
2025-04-04T21:37:32.9622730Z 25/90 Test #25: gamma ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9623510Z       Start 26: gamma_p
2025-04-04T21:37:32.9625010Z Process not started
2025-04-04T21:37:32.9625390Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_gamma_p
2025-04-04T21:37:32.9625730Z [Unknown system error -88]
2025-04-04T21:37:32.9626490Z 26/90 Test #26: gamma_p ..........................***Not Run   0.00 sec
2025-04-04T21:37:32.9627170Z       Start 27: gamma_q
2025-04-04T21:37:32.9628730Z Process not started
2025-04-04T21:37:32.9629010Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_gamma_q
2025-04-04T21:37:32.9629330Z [Unknown system error -88]
2025-04-04T21:37:32.9630260Z 27/90 Test #27: gamma_q ..........................***Not Run   0.00 sec
2025-04-04T21:37:32.9631050Z       Start 28: ligamma
2025-04-04T21:37:32.9632670Z Process not started
2025-04-04T21:37:32.9633070Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_ligamma
2025-04-04T21:37:32.9633410Z [Unknown system error -88]
2025-04-04T21:37:32.9634260Z 28/90 Test #28: ligamma ..........................***Not Run   0.00 sec
2025-04-04T21:37:32.9635140Z       Start 29: log_factorial
2025-04-04T21:37:32.9636940Z Process not started
2025-04-04T21:37:32.9637450Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_log_factorial
2025-04-04T21:37:32.9637800Z [Unknown system error -88]
2025-04-04T21:37:32.9638430Z 29/90 Test #29: log_factorial ....................***Not Run   0.00 sec
2025-04-04T21:37:32.9639240Z       Start 30: log_gamma
2025-04-04T21:37:32.9640860Z Process not started
2025-04-04T21:37:32.9641330Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_log_gamma
2025-04-04T21:37:32.9641700Z [Unknown system error -88]
2025-04-04T21:37:32.9642500Z 30/90 Test #30: log_gamma ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9643400Z       Start 31: uigamma
2025-04-04T21:37:32.9644890Z Process not started
2025-04-04T21:37:32.9645320Z  /Users/runner/work/lfortran/lfortran/stdlib/example/specialfunctions_gamma/example_uigamma
2025-04-04T21:37:32.9645640Z [Unknown system error -88]
2025-04-04T21:37:32.9646550Z 31/90 Test #31: uigamma ..........................***Not Run   0.00 sec
2025-04-04T21:37:32.9647490Z       Start 32: ascii_reverse
2025-04-04T21:37:32.9649140Z Process not started
2025-04-04T21:37:32.9649480Z  /Users/runner/work/lfortran/lfortran/stdlib/example/ascii/example_ascii_reverse
2025-04-04T21:37:32.9649760Z [Unknown system error -88]
2025-04-04T21:37:32.9650650Z 32/90 Test #32: ascii_reverse ....................***Not Run   0.00 sec
2025-04-04T21:37:32.9651520Z       Start 33: ascii_to_lower
2025-04-04T21:37:32.9653030Z Process not started
2025-04-04T21:37:32.9653350Z  /Users/runner/work/lfortran/lfortran/stdlib/example/ascii/example_ascii_to_lower
2025-04-04T21:37:32.9654620Z [Unknown system error -88]
2025-04-04T21:37:32.9654830Z 33/90 Test #33: ascii_to_lower ...................***Not Run   0.00 sec
2025-04-04T21:37:32.9655460Z       Start 34: ascii_to_sentence
2025-04-04T21:37:32.9656930Z Process not started
2025-04-04T21:37:32.9657260Z  /Users/runner/work/lfortran/lfortran/stdlib/example/ascii/example_ascii_to_sentence
2025-04-04T21:37:32.9657560Z [Unknown system error -88]
2025-04-04T21:37:32.9658530Z 34/90 Test #34: ascii_to_sentence ................***Not Run   0.00 sec
2025-04-04T21:37:32.9659280Z       Start 35: ascii_to_title
2025-04-04T21:37:32.9660630Z Process not started
2025-04-04T21:37:32.9660950Z  /Users/runner/work/lfortran/lfortran/stdlib/example/ascii/example_ascii_to_title
2025-04-04T21:37:32.9661230Z [Unknown system error -88]
2025-04-04T21:37:32.9662200Z 35/90 Test #35: ascii_to_title ...................***Not Run   0.00 sec
2025-04-04T21:37:32.9662890Z       Start 36: ascii_to_upper
2025-04-04T21:37:32.9664400Z Process not started
2025-04-04T21:37:32.9664720Z  /Users/runner/work/lfortran/lfortran/stdlib/example/ascii/example_ascii_to_upper
2025-04-04T21:37:32.9665010Z [Unknown system error -88]
2025-04-04T21:37:32.9665940Z 36/90 Test #36: ascii_to_upper ...................***Not Run   0.00 sec
2025-04-04T21:37:32.9666610Z       Start 37: version
2025-04-04T21:37:32.9668100Z Process not started
2025-04-04T21:37:32.9668420Z  /Users/runner/work/lfortran/lfortran/stdlib/example/version/example_version
2025-04-04T21:37:32.9668690Z [Unknown system error -88]
2025-04-04T21:37:32.9669650Z 37/90 Test #37: version ..........................***Not Run   0.00 sec
2025-04-04T21:37:32.9670390Z       Start 38: optval
2025-04-04T21:37:32.9671860Z Process not started
2025-04-04T21:37:32.9672190Z  /Users/runner/work/lfortran/lfortran/stdlib/example/optval/example_optval
2025-04-04T21:37:32.9672460Z [Unknown system error -88]
2025-04-04T21:37:32.9673420Z 38/90 Test #38: optval ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9674140Z       Start 39: check1
2025-04-04T21:37:32.9675800Z Process not started
2025-04-04T21:37:32.9676140Z  /Users/runner/work/lfortran/lfortran/stdlib/example/error/example_check1
2025-04-04T21:37:32.9676420Z [Unknown system error -88]
2025-04-04T21:37:32.9677310Z 39/90 Test #39: check1 ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9678110Z       Start 40: check2
2025-04-04T21:37:32.9679850Z Process not started
2025-04-04T21:37:32.9680150Z  /Users/runner/work/lfortran/lfortran/stdlib/example/error/example_check2
2025-04-04T21:37:32.9680420Z [Unknown system error -88]
2025-04-04T21:37:32.9681420Z 40/90 Test #40: check2 ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9682220Z       Start 41: check3
2025-04-04T21:37:32.9683740Z Process not started
2025-04-04T21:37:32.9684090Z  /Users/runner/work/lfortran/lfortran/stdlib/example/error/example_check3
2025-04-04T21:37:32.9684370Z [Unknown system error -88]
2025-04-04T21:37:32.9685370Z 41/90 Test #41: check3 ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9686220Z       Start 42: check4
2025-04-04T21:37:32.9687700Z Process not started
2025-04-04T21:37:32.9688010Z  /Users/runner/work/lfortran/lfortran/stdlib/example/error/example_check4
2025-04-04T21:37:32.9688270Z [Unknown system error -88]
2025-04-04T21:37:32.9689190Z 42/90 Test #42: check4 ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9689970Z       Start 43: error_stop1
2025-04-04T21:37:32.9691500Z Process not started
2025-04-04T21:37:32.9691810Z  /Users/runner/work/lfortran/lfortran/stdlib/example/error/example_error_stop1
2025-04-04T21:37:32.9692100Z [Unknown system error -88]
2025-04-04T21:37:32.9693010Z 43/90 Test #43: error_stop1 ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9693830Z       Start 44: error_stop2
2025-04-04T21:37:32.9695280Z Process not started
2025-04-04T21:37:32.9695620Z  /Users/runner/work/lfortran/lfortran/stdlib/example/error/example_error_stop2
2025-04-04T21:37:32.9695900Z [Unknown system error -88]
2025-04-04T21:37:32.9696790Z 44/90 Test #44: error_stop2 ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9697570Z       Start 45: chomp
2025-04-04T21:37:32.9699080Z Process not started
2025-04-04T21:37:32.9699410Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_chomp
2025-04-04T21:37:32.9699680Z [Unknown system error -88]
2025-04-04T21:37:32.9700600Z 45/90 Test #45: chomp ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9701330Z       Start 46: count
2025-04-04T21:37:32.9702860Z Process not started
2025-04-04T21:37:32.9703100Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_count
2025-04-04T21:37:32.9703370Z [Unknown system error -88]
2025-04-04T21:37:32.9704480Z 46/90 Test #46: count ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9705390Z       Start 47: ends_with
2025-04-04T21:37:32.9707020Z Process not started
2025-04-04T21:37:32.9707350Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_ends_with
2025-04-04T21:37:32.9707640Z [Unknown system error -88]
2025-04-04T21:37:32.9708510Z 47/90 Test #47: ends_with ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9709260Z       Start 48: find
2025-04-04T21:37:32.9710710Z Process not started
2025-04-04T21:37:32.9711030Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_find
2025-04-04T21:37:32.9711310Z [Unknown system error -88]
2025-04-04T21:37:32.9712290Z 48/90 Test #48: find .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9713060Z       Start 49: padl
2025-04-04T21:37:32.9714550Z Process not started
2025-04-04T21:37:32.9714840Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_padl
2025-04-04T21:37:32.9716030Z [Unknown system error -88]
2025-04-04T21:37:32.9716230Z 49/90 Test #49: padl .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9716780Z       Start 50: padr
2025-04-04T21:37:32.9718200Z Process not started
2025-04-04T21:37:32.9718550Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_padr
2025-04-04T21:37:32.9718820Z [Unknown system error -88]
2025-04-04T21:37:32.9719780Z 50/90 Test #50: padr .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9720520Z       Start 51: replace_all
2025-04-04T21:37:32.9722020Z Process not started
2025-04-04T21:37:32.9722640Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_replace_all
2025-04-04T21:37:32.9722950Z [Unknown system error -88]
2025-04-04T21:37:32.9723550Z 51/90 Test #51: replace_all ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9724360Z       Start 52: slice
2025-04-04T21:37:32.9726020Z Process not started
2025-04-04T21:37:32.9731570Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_slice
2025-04-04T21:37:32.9731860Z [Unknown system error -88]
2025-04-04T21:37:32.9732080Z 52/90 Test #52: slice ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9732330Z Process not started
2025-04-04T21:37:32.9732590Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_starts_with
2025-04-04T21:37:32.9732870Z [Unknown system error -88]
2025-04-04T21:37:32.9733020Z       Start 53: starts_with
2025-04-04T21:37:32.9733220Z 53/90 Test #53: starts_with ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9733440Z       Start 54: strip
2025-04-04T21:37:32.9734250Z Process not started
2025-04-04T21:37:32.9734500Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_strip
2025-04-04T21:37:32.9734780Z [Unknown system error -88]
2025-04-04T21:37:32.9735810Z 54/90 Test #54: strip ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9736620Z       Start 55: to_string
2025-04-04T21:37:32.9738290Z Process not started
2025-04-04T21:37:32.9738660Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_to_string
2025-04-04T21:37:32.9738950Z [Unknown system error -88]
2025-04-04T21:37:32.9739850Z 55/90 Test #55: to_string ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9740660Z       Start 56: zfill
2025-04-04T21:37:32.9742170Z Process not started
2025-04-04T21:37:32.9742510Z  /Users/runner/work/lfortran/lfortran/stdlib/example/strings/example_zfill
2025-04-04T21:37:32.9742780Z [Unknown system error -88]
2025-04-04T21:37:32.9743750Z 56/90 Test #56: zfill ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9744540Z       Start 57: diag1
2025-04-04T21:37:32.9746270Z Process not started
2025-04-04T21:37:32.9746600Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_diag1
2025-04-04T21:37:32.9746870Z [Unknown system error -88]
2025-04-04T21:37:32.9747810Z 57/90 Test #57: diag1 ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9748570Z       Start 58: diag2
2025-04-04T21:37:32.9750190Z Process not started
2025-04-04T21:37:32.9750520Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_diag2
2025-04-04T21:37:32.9750790Z [Unknown system error -88]
2025-04-04T21:37:32.9751830Z 58/90 Test #58: diag2 ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9752670Z       Start 59: diag3
2025-04-04T21:37:32.9754150Z Process not started
2025-04-04T21:37:32.9754490Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_diag3
2025-04-04T21:37:32.9754760Z [Unknown system error -88]
2025-04-04T21:37:32.9755760Z 59/90 Test #59: diag3 ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9756510Z       Start 60: diag4
2025-04-04T21:37:32.9757980Z Process not started
2025-04-04T21:37:32.9758290Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_diag4
2025-04-04T21:37:32.9758550Z [Unknown system error -88]
2025-04-04T21:37:32.9759540Z 60/90 Test #60: diag4 ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9760270Z       Start 61: diag5
2025-04-04T21:37:32.9761700Z Process not started
2025-04-04T21:37:32.9762060Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_diag5
2025-04-04T21:37:32.9762320Z [Unknown system error -88]
2025-04-04T21:37:32.9763210Z 61/90 Test #61: diag5 ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9763960Z       Start 62: eye1
2025-04-04T21:37:32.9765480Z Process not started
2025-04-04T21:37:32.9765820Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_eye1
2025-04-04T21:37:32.9766090Z [Unknown system error -88]
2025-04-04T21:37:32.9767140Z 62/90 Test #62: eye1 .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9767920Z       Start 63: eye2
2025-04-04T21:37:32.9769340Z Process not started
2025-04-04T21:37:32.9769640Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_eye2
2025-04-04T21:37:32.9769900Z [Unknown system error -88]
2025-04-04T21:37:32.9770960Z 63/90 Test #63: eye2 .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9771710Z       Start 64: is_diagonal
2025-04-04T21:37:32.9773230Z Process not started
2025-04-04T21:37:32.9773540Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_diagonal
2025-04-04T21:37:32.9774670Z [Unknown system error -88]
2025-04-04T21:37:32.9774880Z 64/90 Test #64: is_diagonal ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9775530Z       Start 65: is_hermitian
2025-04-04T21:37:32.9777140Z Process not started
2025-04-04T21:37:32.9777450Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_hermitian
2025-04-04T21:37:32.9777730Z [Unknown system error -88]
2025-04-04T21:37:32.9778620Z 65/90 Test #65: is_hermitian .....................***Not Run   0.00 sec
2025-04-04T21:37:32.9779470Z       Start 66: is_hessenberg
2025-04-04T21:37:32.9781120Z Process not started
2025-04-04T21:37:32.9781490Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_hessenberg
2025-04-04T21:37:32.9781780Z [Unknown system error -88]
2025-04-04T21:37:32.9782670Z 66/90 Test #66: is_hessenberg ....................***Not Run   0.00 sec
2025-04-04T21:37:32.9783480Z       Start 67: is_skew_symmetric
2025-04-04T21:37:32.9784880Z Process not started
2025-04-04T21:37:32.9785230Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_skew_symmetric
2025-04-04T21:37:32.9785520Z [Unknown system error -88]
2025-04-04T21:37:32.9786390Z 67/90 Test #67: is_skew_symmetric ................***Not Run   0.00 sec
2025-04-04T21:37:32.9787240Z       Start 68: is_square
2025-04-04T21:37:32.9788740Z Process not started
2025-04-04T21:37:32.9789110Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_square
2025-04-04T21:37:32.9789390Z [Unknown system error -88]
2025-04-04T21:37:32.9790300Z 68/90 Test #68: is_square ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9791130Z       Start 69: is_symmetric
2025-04-04T21:37:32.9792630Z Process not started
2025-04-04T21:37:32.9792990Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_symmetric
2025-04-04T21:37:32.9793270Z [Unknown system error -88]
2025-04-04T21:37:32.9794170Z 69/90 Test #69: is_symmetric .....................***Not Run   0.00 sec
2025-04-04T21:37:32.9794880Z       Start 70: is_triangular
2025-04-04T21:37:32.9796320Z Process not started
2025-04-04T21:37:32.9796670Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_is_triangular
2025-04-04T21:37:32.9796960Z [Unknown system error -88]
2025-04-04T21:37:32.9797810Z 70/90 Test #70: is_triangular ....................***Not Run   0.00 sec
2025-04-04T21:37:32.9798570Z       Start 71: outer_product
﻿2025-04-04T21:37:32.9802640Z Process not started
2025-04-04T21:37:32.9802910Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_outer_product
2025-04-04T21:37:32.9803190Z [Unknown system error -88]
2025-04-04T21:37:32.9803400Z 71/90 Test #71: outer_product ....................***Not Run   0.00 sec
2025-04-04T21:37:32.9803620Z       Start 72: trace
2025-04-04T21:37:32.9804310Z Process not started
2025-04-04T21:37:32.9804680Z  /Users/runner/work/lfortran/lfortran/stdlib/example/linalg/example_trace
2025-04-04T21:37:32.9804940Z [Unknown system error -88]
2025-04-04T21:37:32.9805830Z 72/90 Test #72: trace ............................***Not Run   0.00 sec
2025-04-04T21:37:32.9806700Z       Start 73: arg_select
2025-04-04T21:37:32.9808220Z Process not started
2025-04-04T21:37:32.9808540Z  /Users/runner/work/lfortran/lfortran/stdlib/example/selection/example_arg_select
2025-04-04T21:37:32.9808820Z [Unknown system error -88]
2025-04-04T21:37:32.9809840Z 73/90 Test #73: arg_select .......................***Not Run   0.00 sec
2025-04-04T21:37:32.9810570Z       Start 74: select
2025-04-04T21:37:32.9812120Z Process not started
2025-04-04T21:37:32.9812470Z  /Users/runner/work/lfortran/lfortran/stdlib/example/selection/example_select
2025-04-04T21:37:32.9812750Z [Unknown system error -88]
2025-04-04T21:37:32.9813660Z 74/90 Test #74: select ...........................***Not Run   0.00 sec
2025-04-04T21:37:32.9814460Z       Start 75: clip_integer
2025-04-04T21:37:32.9816160Z Process not started
2025-04-04T21:37:32.9816470Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_clip_integer
2025-04-04T21:37:32.9816750Z [Unknown system error -88]
2025-04-04T21:37:32.9817660Z 75/90 Test #75: clip_integer .....................***Not Run   0.00 sec
2025-04-04T21:37:32.9818380Z       Start 76: clip_real
2025-04-04T21:37:32.9819850Z Process not started
2025-04-04T21:37:32.9820200Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_clip_real
2025-04-04T21:37:32.9820480Z [Unknown system error -88]
2025-04-04T21:37:32.9821420Z 76/90 Test #76: clip_real ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9822170Z       Start 77: diff
2025-04-04T21:37:32.9823630Z Process not started
2025-04-04T21:37:32.9823960Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_diff
2025-04-04T21:37:32.9824220Z [Unknown system error -88]
2025-04-04T21:37:32.9825180Z 77/90 Test #77: diff .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9825860Z       Start 78: gcd
2025-04-04T21:37:32.9827250Z Process not started
2025-04-04T21:37:32.9827550Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_gcd
2025-04-04T21:37:32.9827810Z [Unknown system error -88]
2025-04-04T21:37:32.9828720Z 78/90 Test #78: gcd ..............................***Not Run   0.00 sec
2025-04-04T21:37:32.9829520Z       Start 79: linspace_complex
2025-04-04T21:37:32.9830940Z Process not started
2025-04-04T21:37:32.9831290Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_linspace_complex
2025-04-04T21:37:32.9831580Z [Unknown system error -88]
2025-04-04T21:37:32.9832470Z 79/90 Test #79: linspace_complex .................***Not Run   0.00 sec
2025-04-04T21:37:32.9833290Z       Start 80: linspace_int16
2025-04-04T21:37:32.9834740Z Process not started
2025-04-04T21:37:32.9835040Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_linspace_int16
2025-04-04T21:37:32.9835330Z [Unknown system error -88]
2025-04-04T21:37:32.9836230Z 80/90 Test #80: linspace_int16 ...................***Not Run   0.00 sec
2025-04-04T21:37:32.9836890Z       Start 81: logspace_complex
2025-04-04T21:37:32.9838390Z Process not started
2025-04-04T21:37:32.9838750Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_logspace_complex
2025-04-04T21:37:32.9839040Z [Unknown system error -88]
2025-04-04T21:37:32.9839970Z 81/90 Test #81: logspace_complex .................***Not Run   0.00 sec
2025-04-04T21:37:32.9840690Z       Start 82: logspace_int
2025-04-04T21:37:32.9842230Z Process not started
2025-04-04T21:37:32.9842620Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_logspace_int
2025-04-04T21:37:32.9842890Z [Unknown system error -88]
2025-04-04T21:37:32.9843690Z 82/90 Test #82: logspace_int .....................***Not Run   0.00 sec
2025-04-04T21:37:32.9844450Z       Start 83: logspace_rstart_cbase
2025-04-04T21:37:32.9845850Z Process not started
2025-04-04T21:37:32.9846220Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_logspace_rstart_cbase
2025-04-04T21:37:32.9846590Z [Unknown system error -88]
2025-04-04T21:37:32.9847410Z 83/90 Test #83: logspace_rstart_cbase ............***Not Run   0.00 sec
2025-04-04T21:37:32.9848210Z       Start 84: math_all_close
2025-04-04T21:37:32.9849610Z Process not started
2025-04-04T21:37:32.9849960Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_math_all_close
2025-04-04T21:37:32.9850240Z [Unknown system error -88]
2025-04-04T21:37:32.9851180Z 84/90 Test #84: math_all_close ...................***Not Run   0.00 sec
2025-04-04T21:37:32.9851900Z       Start 85: math_arange
2025-04-04T21:37:32.9853450Z Process not started
2025-04-04T21:37:32.9853780Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_math_arange
2025-04-04T21:37:32.9854050Z [Unknown system error -88]
2025-04-04T21:37:32.9854910Z 85/90 Test #85: math_arange ......................***Not Run   0.00 sec
2025-04-04T21:37:32.9855630Z       Start 86: math_argd
2025-04-04T21:37:32.9857060Z Process not started
2025-04-04T21:37:32.9857390Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_math_argd
2025-04-04T21:37:32.9857660Z [Unknown system error -88]
2025-04-04T21:37:32.9858650Z 86/90 Test #86: math_argd ........................***Not Run   0.00 sec
2025-04-04T21:37:32.9859350Z       Start 87: math_arg
2025-04-04T21:37:32.9860850Z Process not started
2025-04-04T21:37:32.9861140Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_math_arg
2025-04-04T21:37:32.9861400Z [Unknown system error -88]
2025-04-04T21:37:32.9862410Z 87/90 Test #87: math_arg .........................***Not Run   0.00 sec
2025-04-04T21:37:32.9863190Z       Start 88: math_argpi
2025-04-04T21:37:32.9864710Z Process not started
2025-04-04T21:37:32.9865030Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_math_argpi
2025-04-04T21:37:32.9865310Z [Unknown system error -88]
2025-04-04T21:37:32.9866210Z 88/90 Test #88: math_argpi .......................***Not Run   0.00 sec
2025-04-04T21:37:32.9866980Z       Start 89: math_is_close
2025-04-04T21:37:32.9868470Z Process not started
2025-04-04T21:37:32.9868800Z  /Users/runner/work/lfortran/lfortran/stdlib/example/math/example_math_is_close
2025-04-04T21:37:32.9869950Z [Unknown system error -88]
2025-04-04T21:37:32.9870160Z 89/90 Test #89: math_is_close ....................***Not Run   0.00 sec
2025-04-04T21:37:32.9870740Z       Start 90: mean
2025-04-04T21:37:32.9872220Z Process not started
2025-04-04T21:37:32.9872510Z  /Users/runner/work/lfortran/lfortran/stdlib/example/stats/example_mean
2025-04-04T21:37:32.9872770Z [Unknown system error -88]
2025-04-04T21:37:32.9873720Z 90/90 Test #90: mean .............................***Not Run   0.00 sec
2025-04-04T21:37:32.9877020Z 
2025-04-04T21:37:32.9877100Z 0% tests passed, 90 tests failed out of 90
2025-04-04T21:37:32.9877230Z 
2025-04-04T21:37:32.9877280Z Label Time Summary:
2025-04-04T21:37:32.9877440Z quadruple_precision    =   0.00 sec*proc (2 tests)
2025-04-04T21:37:32.9877590Z 
2025-04-04T21:37:32.9877650Z Total Test time (real) =   0.04 sec
2025-04-04T21:37:32.9877760Z 
2025-04-04T21:37:32.9877810Z The following tests FAILED:
2025-04-04T21:37:32.9877970Z 	  1 - logicalloc (BAD_COMMAND)
2025-04-04T21:37:32.9878130Z 	  2 - ascii (BAD_COMMAND)
2025-04-04T21:37:32.9878270Z 	  3 - loadtxt (BAD_COMMAND)
2025-04-04T21:37:32.9878420Z 	  4 - loadtxt_qp (BAD_COMMAND)
2025-04-04T21:37:32.9878580Z 	  5 - savetxt_qp (BAD_COMMAND)
2025-04-04T21:37:32.9878740Z 	  6 - parse_mode (BAD_COMMAND)
2025-04-04T21:37:32.9878890Z 	  7 - linalg (BAD_COMMAND)
2025-04-04T21:37:32.9879170Z 	  8 - linalg_matrix_property_checks (BAD_COMMAND)
2025-04-04T21:37:32.9879360Z 	  9 - optval (BAD_COMMAND)
2025-04-04T21:37:32.9879510Z 	 10 - selection (BAD_COMMAND)
2025-04-04T21:37:32.9879680Z 	 11 - specialfunctions_gamma (BAD_COMMAND)
2025-04-04T21:37:32.9879870Z 	 12 - string_assignment (BAD_COMMAND)
2025-04-04T21:37:32.9880040Z 	 13 - string_operator (BAD_COMMAND)
2025-04-04T21:37:32.9880220Z 	 14 - string_intrinsic (BAD_COMMAND)
2025-04-04T21:37:32.9880440Z 	 15 - string_match (BAD_COMMAND)
2025-04-04T21:37:32.9880610Z 	 16 - string_functions (BAD_COMMAND)
2025-04-04T21:37:32.9880780Z 	 17 - string_strip_chomp (BAD_COMMAND)
2025-04-04T21:37:32.9880990Z 	 18 - string_to_string (BAD_COMMAND)
2025-04-04T21:37:32.9881220Z 	 19 - trapz (BAD_COMMAND)
2025-04-04T21:37:32.9881360Z 	 20 - simps (BAD_COMMAND)
2025-04-04T21:37:32.9881500Z 	 21 - gauss (BAD_COMMAND)
2025-04-04T21:37:32.9881650Z 	 22 - stdlib_math (BAD_COMMAND)
2025-04-04T21:37:32.9882170Z 	 23 - linspace (BAD_COMMAND)
2025-04-04T21:37:32.9882360Z 	 24 - logspace (BAD_COMMAND)
2025-04-04T21:37:32.9882500Z 	 25 - gamma (BAD_COMMAND)
2025-04-04T21:37:32.9882840Z 	 26 - gamma_p (BAD_COMMAND)
2025-04-04T21:37:32.9882990Z 	 27 - gamma_q (BAD_COMMAND)
2025-04-04T21:37:32.9883270Z 	 28 - ligamma (BAD_COMMAND)
2025-04-04T21:37:32.9883420Z 	 29 - log_factorial (BAD_COMMAND)
2025-04-04T21:37:32.9883590Z 	 30 - log_gamma (BAD_COMMAND)
2025-04-04T21:37:32.9883740Z 	 31 - uigamma (BAD_COMMAND)
2025-04-04T21:37:32.9883890Z 	 32 - ascii_reverse (BAD_COMMAND)
2025-04-04T21:37:32.9884050Z 	 33 - ascii_to_lower (BAD_COMMAND)
2025-04-04T21:37:32.9884220Z 	 34 - ascii_to_sentence (BAD_COMMAND)
2025-04-04T21:37:32.9884390Z 	 35 - ascii_to_title (BAD_COMMAND)
2025-04-04T21:37:32.9884560Z 	 36 - ascii_to_upper (BAD_COMMAND)
2025-04-04T21:37:32.9884740Z 	 37 - version (BAD_COMMAND)
2025-04-04T21:37:32.9884900Z 	 38 - optval (BAD_COMMAND)
2025-04-04T21:37:32.9885060Z 	 39 - check1 (BAD_COMMAND)
2025-04-04T21:37:32.9885220Z 	 40 - check2 (BAD_COMMAND)
2025-04-04T21:37:32.9885370Z 	 41 - check3 (BAD_COMMAND)
2025-04-04T21:37:32.9885520Z 	 42 - check4 (BAD_COMMAND)
2025-04-04T21:37:32.9885670Z 	 43 - error_stop1 (BAD_COMMAND)
2025-04-04T21:37:32.9885830Z 	 44 - error_stop2 (BAD_COMMAND)
2025-04-04T21:37:32.9885990Z 	 45 - chomp (BAD_COMMAND)
2025-04-04T21:37:32.9886140Z 	 46 - count (BAD_COMMAND)
2025-04-04T21:37:32.9886290Z 	 47 - ends_with (BAD_COMMAND)
2025-04-04T21:37:32.9886440Z 	 48 - find (BAD_COMMAND)
2025-04-04T21:37:32.9886590Z 	 49 - padl (BAD_COMMAND)
2025-04-04T21:37:32.9886730Z 	 50 - padr (BAD_COMMAND)
2025-04-04T21:37:32.9886870Z 	 51 - replace_all (BAD_COMMAND)
2025-04-04T21:37:32.9887020Z 	 52 - slice (BAD_COMMAND)
2025-04-04T21:37:32.9887190Z 	 53 - starts_with (BAD_COMMAND)
2025-04-04T21:37:32.9887350Z 	 54 - strip (BAD_COMMAND)
2025-04-04T21:37:32.9887490Z 	 55 - to_string (BAD_COMMAND)
2025-04-04T21:37:32.9887660Z 	 56 - zfill (BAD_COMMAND)
2025-04-04T21:37:32.9888160Z 	 57 - diag1 (BAD_COMMAND)
2025-04-04T21:37:32.9888310Z 	 58 - diag2 (BAD_COMMAND)
2025-04-04T21:37:32.9888460Z 	 59 - diag3 (BAD_COMMAND)
2025-04-04T21:37:32.9888600Z 	 60 - diag4 (BAD_COMMAND)
2025-04-04T21:37:32.9888750Z 	 61 - diag5 (BAD_COMMAND)
2025-04-04T21:37:32.9888890Z 	 62 - eye1 (BAD_COMMAND)
2025-04-04T21:37:32.9889030Z 	 63 - eye2 (BAD_COMMAND)
2025-04-04T21:37:32.9889180Z 	 64 - is_diagonal (BAD_COMMAND)
2025-04-04T21:37:32.9889340Z 	 65 - is_hermitian (BAD_COMMAND)
2025-04-04T21:37:32.9889500Z 	 66 - is_hessenberg (BAD_COMMAND)
2025-04-04T21:37:32.9890020Z Errors while running CTest
2025-04-04T21:37:32.9890390Z Output from these tests are in: /Users/runner/work/lfortran/lfortran/stdlib/Testing/Temporary/LastTest.log
2025-04-04T21:37:32.9890860Z Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
2025-04-04T21:37:32.9891140Z 	 67 - is_skew_symmetric (BAD_COMMAND)
2025-04-04T21:37:32.9891310Z 	 68 - is_square (BAD_COMMAND)
2025-04-04T21:37:32.9891470Z 	 69 - is_symmetric (BAD_COMMAND)
2025-04-04T21:37:32.9891920Z 	 70 - is_triangular (BAD_COMMAND)
2025-04-04T21:37:32.9892220Z 	 71 - outer_product (BAD_COMMAND)
2025-04-04T21:37:32.9892450Z 	 72 - trace (BAD_COMMAND)
2025-04-04T21:37:32.9892610Z 	 73 - arg_select (BAD_COMMAND)
2025-04-04T21:37:32.9892780Z 	 74 - select (BAD_COMMAND)
2025-04-04T21:37:32.9892930Z 	 75 - clip_integer (BAD_COMMAND)
2025-04-04T21:37:32.9893100Z 	 76 - clip_real (BAD_COMMAND)
2025-04-04T21:37:32.9893250Z 	 77 - diff (BAD_COMMAND)
2025-04-04T21:37:32.9893490Z 	 78 - gcd (BAD_COMMAND)
2025-04-04T21:37:32.9893650Z 	 79 - linspace_complex (BAD_COMMAND)
2025-04-04T21:37:32.9893840Z 	 80 - linspace_int16 (BAD_COMMAND)
2025-04-04T21:37:32.9894020Z 	 81 - logspace_complex (BAD_COMMAND)
2025-04-04T21:37:32.9894200Z 	 82 - logspace_int (BAD_COMMAND)
2025-04-04T21:37:32.9894370Z 	 83 - logspace_rstart_cbase (BAD_COMMAND)
2025-04-04T21:37:32.9894550Z 	 84 - math_all_close (BAD_COMMAND)
2025-04-04T21:37:32.9894740Z 	 85 - math_arange (BAD_COMMAND)
2025-04-04T21:37:32.9894990Z 	 86 - math_argd (BAD_COMMAND)
2025-04-04T21:37:32.9895150Z 	 87 - math_arg (BAD_COMMAND)
2025-04-04T21:37:32.9895300Z 	 88 - math_argpi (BAD_COMMAND)
2025-04-04T21:37:32.9900830Z 	 89 - math_is_close (BAD_COMMAND)
2025-04-04T21:37:32.9901040Z 	 90 - mean (BAD_COMMAND)
2025-04-04T21:37:32.9905920Z ##[error]Process completed with exit code 8.
```

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -795,9 +795,6 @@ jobs:
             make -j8
             ctest
 
-            # Test with GFortran
-            ./build_test_gf.sh
-
       - name: Test SNAP
         shell: bash -e -x -l {0}
         run: |

